fhir-server-test/src/test/resources/test.properties

```diff
@@ -60,5 +60,6 @@ test.bulkdata.export.s3.enabled = false
 # Enable/disable the Azure storage provider tests
 test.bulkdata.import.azure.enabled = false
 test.bulkdata.export.azure.enabled = false
-test.audit.kafka.connectionInfo = localhost:29094,localhost:49098
-test.audit.kafka.topicName = FHIR_AUDIT
```

Comment:
`test.audit.kafka.topicName` isn't needed in this properties file any more?

Reply:
ah, i see we get it from the fhir-server-config now (via ConfigurationTranslator); great